src/PIL/PdfImagePlugin.py

```diff
@@ -123,8 +124,26 @@ def _save(im, fp, filename, save_all=False):
             params = None
             decode = None
 
+            #
+            # Get image characteristics
+
+            width, height = im.size
+
             if im.mode == "1":
-                filter = "DCTDecode"
+                filter = "CCITTFaxDecode"
+                bits = 1
+                params = PdfParser.PdfArray(
+                    [
+                        PdfParser.PdfDict(
+                            {
+                                "K": -1,
+                                "BlackIs1": True,
+                                "Columns": width,
+                                "Rows": height,
+                            }
+                        )
+                    ]
+                )
```

Comment:
https://opensource.adobe.com/dc-acrobat-sdk-docs/standards/pdfstandards/pdf/PDF32000_2008.pdf#page=38 indicates that K should be <0 for Group 4 encoding

On the next page of that document, it describes BlackIs1, that allows the colors to be inverted.